examples/launch_height_sagemaker_checkpoints.py

```diff
@@ -13,17 +13,18 @@
 import logging
 from pathlib import Path
 
-from sagemaker.sklearn import SKLearn
-
+from syne_tune import Tuner, StoppingCriterion
 from syne_tune.backend import SageMakerBackend
+from syne_tune.backend.sagemaker_backend.estimators import (
+    sklearn_estimator,
```

Comment:
It would be better to use `basic_cpu_instance_sagemaker_estimator` here

Reply:
Done